e2e/pnpm_lockfiles/v90/pnpm-lock.yaml

```diff
@@ -181,6 +184,24 @@ importers:
         specifier: link:../b
         version: link:../b
 
+  ../projects/peer-types:
+    dependencies:
+      '@scoped/a':
+        specifier: workspace:*
+        version: link:../a
+      '@scoped/b':
+        specifier: link:../b
+        version: link:../b
+      '@scoped/c':
+        specifier: file:../c
+        version: file:../projects/c(@scoped/b@projects+b)
+      hello:
+        specifier: https://gitpkg.vercel.app/EqualMa/gitpkg-hello/packages/hello
+        version: https://gitpkg.vercel.app/EqualMa/gitpkg-hello/packages/hello
+      jsonify:
+        specifier: https://github.com/aspect-build/test-packages/releases/download/0.0.0/@foo-jsonify-0.0.0.tgz
+        version: '@foo/jsonify@https://github.com/aspect-build/test-packages/releases/download/0.0.0/@foo-jsonify-0.0.0.tgz'
```

Comment:
Notice the `://` here. This shows a difference in v9+ where the version being a URL prefixes the `pkgname@` unlike <v9.

This is what required the new `replace("://", "/")` (which I didn't realize was in this cherry-pick 😅)